frontend/ui/src/components/organization/tree-item-row.tsx

```
@@ -0,0 +1,325 @@
+          )}
+        </TableCell>
+        <TableCell className="border border-gray-200">
+          {/* Created on would go here */}
+        </TableCell>
+        <TableCell className="border border-gray-200">
+          {/* Created by would go here */}
+        </TableCell>
+        <TableCell className="w-10 border border-gray-200">
+          <DropdownMenu.Root>
+            <DropdownMenu.Trigger>
+              <button className="rounded-md p-1 hover:bg-gray-100 focus:outline-none">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)